src/inc/dacprivate.h

```diff
@@ -20,6 +20,14 @@
 #include <msodw.h>
 #endif // FEATURE_PAL
 
+#if !defined(MSLAYOUT)
```

Comment:
I wonder if there could be a common place to put this macro. I am not thrilled by having the same definition at two places - here and in the src/debug/inc/common.h.